project_task_note/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module add notes in project task only visible for internal users
```

Comment:
```suggestion
This module add notes in project tasks, only visible for internal users.
```

Add also a USAGE section for indicating where to see that notes, and maybe a CONTEXT file for indicating why we are creating this (because there's no field where storing general comments and notes not visible to the customers in the portal).